addons/mail/static/src/rtc/call_action_list.js

```diff
@@ -51,6 +69,21 @@ export class CallActionList extends Component {
         }
     }
 
+    onClickDownloadLogs() {
+        const data = window.JSON.stringify(Object.fromEntries(this.rtc.state.logs));
+        const blob = new window.Blob([data], { type: "application/json" });
+        const url = window.URL.createObjectURL(blob);
+        const a = document.createElement("a");
+        a.href = url;
+        a.download = `RtcLogs_Channel_${this.rtc.state.logs.get(
+            "channelId"
+        )}_Session_${this.rtc.state.logs.get("selfSessionId")}_${luxon.DateTime.now().toFormat(
+            "yyyy-ll-dd_HH-mm"
+        )}.json`;
```

Comment:
I feel like this template literal is getting a bit too intricate. For the sake of readability, it might be worth moving its computed parts into well named variables.